pkgs/development/python-modules/confusable-homoglyphs/default.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pytestCheckHook,
+  pythonOlder,
+  setuptools,
+  click,
+}:
+
+let
+  name = "confusable-homoglyphs";
+  snakeName = builtins.replaceStrings [ "-" ] [ "_" ] name;
+in
+buildPythonPackage rec {
+  pname = name;
```

Comment:
This variable name should not be used because `name` is an internal attribute.
https://github.com/NixOS/nixpkgs/blob/189e5f171b163feb7791a9118afa778d9a1db81f/pkgs/stdenv/generic/make-derivation.nix#L333-L361

Reply:
addressed